aidants_connect_web/views/service.py

```diff
@@ -36,14 +36,20 @@
 log = logging.getLogger()
 
 
-def humanize_demarche_names(list_of_machine_names):
-    human_names = []
-    for p in list_of_machine_names:
-        for category in settings.DEMARCHES:
-            for element in category[1]:
-                if element[0] == p:
-                    human_names.append(element[1])
-    return human_names
+def humanize_demarche_names(machine_names: list) -> list:
+    """
```

Comment:
I'm not familiar with this syntax, could you tell me more about it?

Reply:
It is a way to inform about and enforce types in python.
```py
machine_names: list
```
means that the argument `machine_names` is a list

```py
-> list
```
means that the function returns a list